chapters/functions.tex

```diff
@@ -2579,7 +2579,7 @@ \subsubsection{Input Parameters, Function Value}\label{input-parameters-function
 \end{lstlisting}
 \end{example}
 
-\subsubsection{Arbitrary Placement of Output Parameters, No External Function Value}\label{arbitrary-placement-of-output-parameters-no-external-function-value}
+\subsubsection{Arbitrary Placement of Output Parameters, No External Function Value}\label{arbitrary-placement-of-output-parameters-no-external-function-value}\label{both-function-value-and-output-variable}
```

Comment:
Is this label in the right place? The subsubsection called Both Function Value and Output Variable is on line 2612.

Reply:
oops - no